docs/getting-started.md

```diff
@@ -1,21 +1,180 @@
+
+#### Send a Private Message to Someone New
+
+You can send a PM by moving your cursor to the list of "Users" in the left column and selecting the name of the person you'd like to send a message to.
+
+If you grabbed the latest unreleased version from [Github](https://github.com/zulip/zulip-terminal) or are running a version greater than 0.6.0 (use `zulip-term --version` to check), you can try out our new autocomplete feature! 
+
+To send a PM using the autocomplete feature:
+1. Type <kbd>x</kbd> from anywhere in Zulip Terminal. A message editor will pop open at the bottom of the middle column. 
+2. Type in part of the name of the person you'd like to send a PM to (IMG 1)
+3. Press <kbd>ctrl</kbd><kbd>f</kbd> and a list of potential recipients will appear in the footer (highlighted in red). 
+4. Press <kbd>ctrl</kbd><kbd>f</kbd> until the name of the person you want to send a message to is highlighted (IMG 2). 
+5. Press <kbd>tab</kbd> to jump to the message section and type in your message. 
+6. Press <kbd>ctrl</kbd><kbd>d</kbd> to send.
+7. Press <kbd>esc</kbd> to get out of the message editor.
+
+Try following the steps above to send a message to the Welcome Bot!
+
+*You must use <kbd>ctrl</kbd><kbd>f</kbd> to autocomplete the name and zulip address of the person you want to PM. If you just type in their full name Zulip Terminal will throw an error.*
+
+<img src="getting-started-imgs/send-PM1.png" width="75%">
+<img src="getting-started-imgs/send-PM2.png" width="75%">
+
+### Create a New Topic
+
+To create a new topic in a stream, type <kbd>c</kbd> and a message editor will pop open at the bottom of the middle column. Make sure your cursor is in the middle column, otherwise nothing will happen when you type <kbd>c</kbd>.
+
+By default this will send a new message to the stream of whichever message you have selected. Let's send a test message to [# test here](https://chat.zulip.org/#narrow/stream/7-test-here). If you're not already in the [# test here](https://chat.zulip.org/#narrow/stream/7-test-here) stream, replace the stream name with 'test here'. Use <kbd>tab</kbd> to jump to the topic section and type in a topic. Type <kbd>tab</kbd> again to jump to the message section and type in a message. Type <kbd>ctrl</kbd><kbd>d</kbd> to send. If you're following along, feel free to actually send the message, [# test here](https://chat.zulip.org/#narrow/stream/7-test-here) was for things like this tutorial.
+
+Type <kbd>esc</kbd> to get out of the message editor.
+
+<img src="getting-started-imgs/new-topic.png" width="75%">
+
+## Edit a Message
+
+Let's try editing the new topic we added to the [# test here](https://chat.zulip.org/#narrow/stream/7-test-here) stream. By default, Zulip allows you to edit the content of your messages within 10 minutes of when you send them.
```

Comment:
This 10m default is good, but if on chat.zulip.org (or other servers) these values may be different, including for topics - eg. it's currently 1h on chat.zulip.org for content. It may be too complex, but we could link to https://chat.zulip.org/help/configure-message-editing-and-deletion .

Reply:
I think the article is a bit out of the league of this tutorial given that the feature is only accessible by administrators. However I did add some clarification about the editing time.